src/pages/ReportSettingsPage.js

```diff
@@ -84,12 +97,58 @@ class ReportSettingsPage extends Component {
 
         this.state = {
             newRoomName: this.props.report.reportName,
-            error: '',
+            errors: {},
         };
+
+        this.validateAndRenameReport = this.validateAndRenameReport.bind(this);
+    }
+
+    validateAndRenameReport() {
+        if (!this.validate() || this.props.report.reportName === this.state.newRoomName) {
```

Comment:
It seems that showing a successful growl instead of skipping when the user uses the same name is better UX. Could you please update that? It is also mentioned on the slack.

Reply:
Without making an API call, just show the success growl.